### ISSUE TITLE
Error message confusing if glyph name is "A" or "a"

### ISSUE THREAD

**Issue by madig** [source-foundry/ufolint#25]
```Python
f=defcon.Font()
f.newGlyph("a")
f.save("test.ufo")
```
```
> echo "asdasdasfdfsdfsdg<<<<<<<<<<<<<>>>>>>>>>>>>>" >> test.ufo/glyphs/a.glif
> ufolint test.ufo

ufolint:
~~~~~~~~~~
.\test.ufo
~~~~~~~~~~

[UFO directory] .......
   Found UFO v3
   Detected glyphs directories:
     -- glyphs .
[UFO v3 mandatory files] ...
[XML formatting] ........
[*.plist spec] ........
[features.fea] not present
[data] not present
[images] not present
[*.glif spec]
 - .\test.ufo\glyphs  F   1 .glif tests completed


==========================================================
.\test.ufo failed ufolint testing! Exit with status code 1
==========================================================

[FAIL] a test failed with error: Extra content at the end of the document, line 6, column 1 (<string>, line 6)
```
When a UFO has 1000 glyphs, this gets guessy quickly ;)

**Comment by madig** [comment]
Oh wait. It does! "**a** test failed". I tried this with the `A` and `a` glyphs before and apparently got confused 😁

**Comment by chrissimpkins** [comment]
Yep we need the path to the failing file.  Will look into it.  

For current indicators there is the stderr message at the end of the report and the `F` under the.glif spec tests.

**Comment by madig** [comment]
Hm. I wonder if it catches errors in different layers. *goes off to test*

**Comment by chrissimpkins** [comment]
@madig your changes in #26 were merged and pushed to PyPI as v0.3.5. Thank you!!
